src/styles/main.scss

```
@@ -1,3 +1,98 @@
+$days: mon, tue, wed, thu, fri, sat, sun;
+$width-of-date: 100px;
+$height-of-date: 100px;
+
+$dayMargins: (
+  mon: 0,
+  tue: 101px,
+  wed: 202px,
+  thu: 303px,
+  fri: 404px,
+  sat: 505px,
+  sun: 606px
+);
```

you should calculate this values at this loop 
example:
```
  @each $day, $number in $days {
    &--start-day--#{$day} :first-child {
      margin-left: ($cell-size + $gap-size) * $number;
    }
  }
```

```suggestion
$dayMargins: (
  mon: 0,
  tue: 1,
  wed: 2,
  thu: 3,
  fri: 4,
  sat: 5,
  sun: 6
);
```